File: files/INSPINIA_v4.2.0/ASP.Net-MVC5/Full/Controllers/PagesController.cs
using Microsoft.AspNetCore.Mvc;

namespace Inspinia.Controllers
{
    public class PagesController : Controller
    {
        public IActionResult ComingSoon() => View();
        public IActionResult EmptyPage() => View();
        public IActionResult Faq() => View();
        public IActionResult Pricing() => View();
        public IActionResult Profile() => View();
        public IActionResult SearchResults() => View();
        public IActionResult TermsConditions() => View();
        public IActionResult Timeline() => View();
    }
}

File: files/INSPINIA_v4.2.0/RoR/Full/app/controllers/pages_controller.rb
class PagesController < ApplicationController
    layout :choose_layout

    def choose_layout
       action_name.in?(%w[maintenance coming_soon]) ? "base" : "vertical"
    end

    def profile
    end

    def faq
    end 

    def pricing
    end

    def empty_page
    end

    def timeline
    end

    def search_results
    end

    def coming_soon
    end

    def terms_conditions
    end
end
